felt/src/lib.rs

```diff
@@ -897,17 +897,17 @@ mod test {
 
         #[test]
         #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
-        fn to_be_bytes(ref x in FELT_PATTERN) {
-            let x = &Felt252::from_bytes_be(x.as_bytes());
+        fn to_be_bytes(ref x in "[1-9][0-9]*") {
```

Comment:
```suggestion
        fn to_be_bytes(ref x in FELT_NON_ZERO_PATTERN) {
```

Reply:
Nice catch! I changed the strategy anyway tho.